VideoWeb/VideoWeb/Controllers/UserDataController.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using VideoWeb.Contract.Request;
+using VideoWeb.Contract.Responses;
+using VideoWeb.Services.User;
+
+namespace VideoWeb.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/accounts")]
+    [ApiController]
+    public class UserDataController : ControllerBase
+    {
+        private readonly IUserApiClient _userApiClient;
+        private readonly ILogger<UserDataController> _logger;
+
+        public UserDataController(IUserApiClient userApiClient, ILogger<UserDataController> logger)
+        {
+            _userApiClient = userApiClient;
+            _logger = logger;
+        }
+
+        /// <summary>
+        /// Get Court rooms accounts (judges)
+        /// </summary>
+        [HttpGet("courtrooms", Name = "GetCourtRoomAccounts")]
+        [ProducesResponseType(typeof(IList<CourtRoomsAccountResponse>), (int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        public async Task<ActionResult<IList<CourtRoomsAccountResponse>>> GetCourtRoomsAccounts([FromQuery]VhoConferenceFilterQuery query)
+        {
+            try
+            {
+                var response = await _userApiClient.GetJudgesAsync();
+
+                var accountList = response.Where(x => query.UserNames.Any(s => x.First_name == s))
```

Comment:
Please move this into a mapper to maintain consistency and keep the controller as lean as possible

Reply:
Done